docs/home_assistant.md

```diff
@@ -41,19 +41,6 @@ Delete automatically installed version:
 rm .homeassistant/deps/lib/python3.5/site-packages/xknx*
 ```
 
-Comment out dependency from xknx.py, otherwise hass would reinstall the xknx library when startup:
-
-````diff
-julius@xxx ~/xknx:$ git diff -U0 home-assistant-plugin/custom_components/xknx.py 
-diff --git a/home-assistant-plugin/custom_components/xknx.py b/home-assistant-plugin/custom_components/xknx.py
-index 7f26c43..0b1e046 100644
---- a/home-assistant-plugin/custom_components/xknx.py
-+++ b/home-assistant-plugin/custom_components/xknx.py
-@@ -39 +39 @@ _LOGGER = logging.getLogger(__name__)
--REQUIREMENTS = ['xknx==0.7.18']
-+# REQUIREMENTS = ['xknx==0.7.18']
-```
-
```

Comment:
This isn't part of home-assistant-plugin/custom_components/xknx.py anymore. 
It loaded my checked out version of xknx without removing the requirement from manifest.json. Can someone confirm this?

Reply:
You need a current version of HASS. They now read the dependencies from manifest: https://github.com/home-assistant/home-assistant/pull/22699